src/program/db/db_functions.py

```diff
@@ -321,27 +259,16 @@ def load_streams_in_pages(session: Session, media_item_id: int, page_number: int
         stream = session.query(Stream).get(stream_id)
         yield stream_id, infohash, stream
 
-def _get_item_ids(session, item):
-    from program.media.item import Episode, Season
-
-    if item.type == "show":
-        show_id = item._id
-
-        # season_alias = aliased(Season, flat=True)
-        # season_query = select(Season._id.label('id')).where(Season.parent_id == show_id)
-        # episode_query = (
-        #     select(Episode._id.label('id'))
-        #     .join(season_alias, Episode.parent_id == season_alias._id)
-        #     .where(season_alias.parent_id == show_id)
-        # )
+def _get_item_ids(session, item_id: int) -> tuple[int, list[int]]:

```

Comment:
This looks like it can be useful, but a majority of the time we are just looking at item.type and not much else 🤔 

I'll see if I can use load_only in some spots. Tons of info on that page you linked though, looking over it now